### PR TITLE
Cleanup loss function

### PR DESCRIPTION
**Pull request by Richienb**

### CODE REVIEW

File: loss.py [hunk 1]
```diff
@@ -7,35 +7,38 @@ class PULoss(nn.Module):
...
-        positive_risk = self.prior * torch.sum(y_positive)/ n_positive
-        negative_risk = - self.prior *torch.sum(y_positive_inv)/ n_positive + torch.sum(y_unlabeled)/n_unlabeled
+        positive, unlabeled = target == self.positive, target == self.unlabeled
+        positive, unlabeled = positive.type(torch.float), unlabeled.type(torch.float)
+
+        n_positive, n_unlabeled = torch.clamp(torch.sum(positive), min=self.min_count), torch.clamp(torch.sum(unlabeled), min=self.min_count)
+
+        y_positive = self.loss_func(inp) * positive
+        y_positive_inv = self.loss_func(-inp) * positive
+        y_unlabeled = self.loss_func(-inp) * unlabeled
```
Comment (Richienb) [inline loss.py:36]: `inp` doesn't need to be multiplied by the mask because that is already done outside of the function.
